guide/src/ecosystem/async-await.md

```diff
@@ -12,6 +12,176 @@ In the following sections, we'll give a general overview of `pyo3-asyncio` expla
 async Python functions with PyO3, how to call async Rust functions from Python, and how to configure
 your codebase to manage the runtimes of both.
 
+## Quickstart
+
+Here are some examples to get you started right away! A more detailed breakdown
+of the concepts in these examples can be found in the following sections.
+
+### Rust Applications
+Here we initialize the runtime, import Python's `asyncio` library and run the given future to completion using Python's default `EventLoop` and `async-std`. Inside the future, we convert `asyncio` sleep into a Rust future and await it.
+
+
+```toml
+# Cargo.toml dependencies
+[dependencies]
+pyo3 = { version = "0.14" }
+pyo3-asyncio = { version = "0.14", features = ["attributes", "async-std-runtime"] }
+async-std = "1.9"
+```
+
+```rust
+//! main.rs
+
+use pyo3::prelude::*;
+
+#[pyo3_asyncio::async_std::main]
+async fn main() -> PyResult<()> {
```

Comment:
Yes, `pyo3_asyncio::async_std::main` calls `pyo3::prepare_freethreaded_python`. I mentioned that in the migration guide, but maybe it should be in the examples too. 

The reason I did that rather than require `auto-initialize` is because I didn't want to have that feature automatically enabled when people use `pyo3-asyncio` as a dependency (assuming they aren't using the proc macros).